packages/broker/src/plugins/storage/StoragePlugin.ts

```diff
@@ -21,6 +21,11 @@ export interface StoragePluginConfig {
     storageConfig: {
         refreshInterval: number
     }
+    cluster: {
+        clusterAddress: string,
```

Comment:
When would the `clusterAddress` be null here?

Reply:
@harbu the logic is that if `clusterAddress` is null (which it is by default), the the address of the node is used. I'm not sure if it's possible to configure such logic in the json schema and/or TS file?

The idea was that the default values should result in a singleton storage node (= a cluster of size 1, subscribing to streams assigned to the node's own address).

Reply:
Added comment about it

Reply:
Ah got it!